Added this.name in replace of New Watch string - isNew Create else Save

jkelastic opened the pull request:
## Summary

Fixes #60700 Replaced 'New Watch' with this.name to display the saved watch name

### Release note
This PR fixes the copy of the success notification that displays after creating or saving a watch.

elasticmachine commented:
Pinging @elastic/es-ui (Team:Elasticsearch UI)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65868/)
* Commit: de25c89eb264c01658ba5cf8b582b21e04cd0f0a
* This comment will update when the build is complete

### Failed CI Steps
* [Execute kibana-intake](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65868/execution/node/308/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65868/","number":"65868","commit":"de25c89eb264c01658ba5cf8b582b21e04cd0f0a"}]}
PIPELINE-->

jkelastic commented:
@alisonelizabeth After addressing the merge and adding back ```xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription``` I get the same error. Not sure why that is related to my code. What do you advise?


```

xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription
ERROR Unhandled exception!
ERROR Error: 
      1 unused translation(s):

```

alisonelizabeth commented on x-pack/plugins/translations/translations/ja-JP.json:
```diff
@@ -15238,8 +15238,8 @@
     "xpack.securitySolution.auditd.executionOfForbiddenProgramDescription": "禁止されたプログラムの実行",
-    "xpack.securitySolution.auditd.failedLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
-    "xpack.securitySolution.auditd.inDescription": "in",
+    "xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
```
I don't think this line needs to change. Looks like there is a typo now in the translation id.

jkelastic replied:
thanks for the second pair of eyes

alisonelizabeth commented on x-pack/plugins/translations/translations/ja-JP.json:
```diff
@@ -15238,8 +15238,8 @@
-    "xpack.securitySolution.auditd.failedLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
-    "xpack.securitySolution.auditd.inDescription": "in",
+    "xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription": "ログイン回数超過によりログインに失敗",
+    "xpack.securitySolution.auditd.inDescription": "次に含まれる：",
```
Did you translate this manually? I think we should probably leave it how it exists on `master`.

jkelastic replied:
I didn't translate this manually. I think when I tried to remove `xpack.securitySolution.auditd.failedxLoginTooManyTimesDescription` initially and did some cut and paste I must have replaced it accidentally.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65993/)
* Commit: e35cf6fadc2f9760bfd95c6b3239038fa75a7e9d

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/e35cf6fadc2f9760bfd95c6b3239038fa75a7e9d) | diff | [baseline](https://github.com/elastic/kibana/commit/1c0b77c494c7d0a85a922f84597a5c3b2ecd4da8) |
| --- | --- | --- | --- |
| watcher | 604.9KB | +73.0B | 604.8KB |

### History
* :broken_heart: [Build #65868](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65868/) failed de25c89eb264c01658ba5cf8b582b21e04cd0f0a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65868/","number":"65868","commit":"de25c89eb264c01658ba5cf8b582b21e04cd0f0a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65993/","number":"65993","commit":"e35cf6fadc2f9760bfd95c6b3239038fa75a7e9d"}]}
PIPELINE-->

jkelastic commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65994/)
* Commit: d321fbe38764e630c9e274b9799b24097c7b0b06

### Build metrics
#### async chunks size
| id | [value](https://github.com/elastic/kibana/commit/d321fbe38764e630c9e274b9799b24097c7b0b06) | diff | [baseline](https://github.com/elastic/kibana/commit/2d2e959f5d765fb6a0bbd7cfd1602b7bfb0f12e9) |
| --- | --- | --- | --- |
| watcher | 604.9KB | +73.0B | 604.8KB |

### History
* :green_heart: [Build #65993](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65993/) succeeded e35cf6fadc2f9760bfd95c6b3239038fa75a7e9d
* :broken_heart: [Build #65868](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65868/) failed de25c89eb264c01658ba5cf8b582b21e04cd0f0a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65868/","number":"65868","commit":"de25c89eb264c01658ba5cf8b582b21e04cd0f0a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65993/","number":"65993","commit":"e35cf6fadc2f9760bfd95c6b3239038fa75a7e9d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/65994/","number":"65994","commit":"d321fbe38764e630c9e274b9799b24097c7b0b06"}]}
PIPELINE-->

jkelastic commented:
> LGTM. Tested latest changes and verified the toast notification displays the correct text. Nice job @jkelastic! Thanks for working on this one.
> 
> I updated the PR description and added a release note. Once the build is green, can you merge to master and then also backport to `7.x`? Thanks!

@alisonelizabeth No problem, happy and having fun working on this . Thank for all the help.
